src/containerloops.jl

```diff
@@ -0,0 +1,240 @@
+import Base.keys
+import Base.values
+
+## These are the containers that can be looped over
+## The suffix 0 is for SortedDict and SortedMultiDict
+## The suffix 1 is for SortedSet.  No suffix means any of the
+## three.  
+
+typealias SCContainers0 Union(SortedDict, SortedMultiDict)
+typealias SCContainers Union(SCContainers0, SortedSet)
```

Comment:
I suppose it stands for Sorted Container, which is a bit redundant, isn't 
it?  I can fix this name.

-- Steve


On Fri, 29 May 2015, Kevin Squire wrote:

> 
> In src/containerloops.jl:
> 
> > @@ -0,0 +1,240 @@
> > +import Base.keys
> > +import Base.values
> > +
> > +## These are the containers that can be looped over
> > +## The suffix 0 is for SortedDict and SortedMultiDict
> > +## The suffix 1 is for SortedSet.  No suffix means any of the
> > +## three. 
> > +
> > +typealias SCContainers0 Union(SortedDict, SortedMultiDict)
> > +typealias SCContainers Union(SCContainers0, SortedSet)
> 
> What does SC stand for?
> 
> —
> Reply to this email directly or view it on GitHub.[AHhEh8nA6KxKxoW-hT2ff45Uu3MfoInQks5oOUKfgaJpZM4EMDTc.gif]
> 
> 
>